src/test/java/picard/analysis/CollectMultipleMetricsTest.java

```diff
@@ -335,8 +335,10 @@ void setupBuilder() throws IOException {
         final String flowCellBarcode = "TESTBARCODE";
 
         tempSamFile = File.createTempFile("CollectGcBias", ".bam", TEST_DIR);
+        final File tempSamIndex = new File(tempSamFile.getAbsolutePath().replace("bam", "bai"));
```

Comment:
This test would successfully clean up after itself when run alone with `-Dtest.single=CollectMultipleMetricsTest` but fail to clean up when run with all tests in the `analysis` package (or all the tests in Picard). Explicitly marking the index file for deletion fixes the problem, but I'm not sure what the root issue is.